src/instagram.js

```diff
@@ -90,10 +90,11 @@ export async function apiInstagramPosts({
   access_token,
   instagram_id,
   username,
+  limit = '100',
```

Comment:
i haven't added automatic linter but project is using backticks for strings. eslint should warn you